editor/plugins/script_text_editor.cpp

```diff
@@ -1598,8 +1598,27 @@ static Node *_find_script_node(Node *p_edited_scene, Node *p_current_node, const
 	return nullptr;
 }
 
+
+String ScriptTextEditor::_escape_drop_data(const String &str) {
```

Comment:
Could we make this static?

Reply:
Done! No reason it shouldn't be indeed :)